job_spec/WATER_MAP.yml

```
@@ -94,6 +94,14 @@ WATER_MAP:
         description: Maximum bound used for iterative method. Ignored when flood_depth_estimator is None.
         default: 15
         type: integer
+    minimzation_metric:
+      api_schema:
+        description: Evaluation method to minimize. A value of None indicates that flood depth estimation will not be performed.
```

Is this correct? (I don't actually know.)
```suggestion
        description: Evaluation method to minimize. Ignored when flood_depth_estimator is None.
```

---

Oh gosh yea that's not right. I think we'll want to say something like "A threshold level will be computed if value is None"